Add PixelPen in the list of 4.2+ godot plugins (#340)
godotengine/awesome-godot: README.md

- [Maaack's Game Template Plugin](https://github.com/Maaack/Godot-Game-Template) - Template with a main menu, options menus, pause menu, credits, scene loader, extra tools, and an example game scene.
- [Maaack's Menus Template](https://github.com/Maaack/Godot-Menus-Template) - Template with a main menu, options menus, credits, and a scene loader.
- [Maaack's Scene Loader](https://github.com/Maaack/Godot-Scene-Loader) - Scene loader that includes a loading screen, progress bar, and error handling.

> - [PixelPen](https://github.com/pixelpen-dev/pixelpen) - PixelArt Drawing and Animation: Includes common drawing tools such as selection, shape creation, mirroring, tiling, and animation features. All in godot editor. 

Reviewer: ```suggestion
- [PixelPen](https://github.com/pixelpen-dev/pixelpen) - Pixel art drawing and animation: Common drawing tools such as selection, shape creation, mirroring, tiling, and animation features within the Godot editor.
```
